### PR TITLE
Creating new repo libavtp with same commit history

### PR DESCRIPTION
**Pull request by avinash-palleti**
As part of OpenAVnu restructuring removing libavtp folder from OpenAVnu and created as separate repo with same commit history.

### CODE REVIEW

**Comment by andrew-elder** [comment]
Is there a CI step to set up?

**Comment by avinash-palleti** [comment]
Will submit travis.yml file to build libavtp but CI infrastructure needs to be updated to consider this new repo.

**Comment by andrew-elder** [comment]
@avinash-palleti - are you saying you would like this pull request accepted without CI support? Or, are you planning an additional commit?

**Comment by avinash-palleti** [comment]
I will add another commit to this PR only which adds .travis.yml file.
But that will not be enough, CI infrastructure also needs to be updated to consider this repo.

**Comment by aguedes** [comment]
Hi @avinash-palleti, I see you added your commit 8d64522 to this PR but I think it should at least preserve the commit message from the "original" commit https://github.com/AVnu/OpenAvnu/commit/76c6e71e2a56ba19b4527abebc24508d3e0bce76.

Also it looks like the .travis.yml file needs some clean up. For instance, "BUILD_KERNEL=4.4.0-75-generic" doesn't seem to be actually required.

Finally, this PR is missing the commit https://github.com/AVnu/OpenAvnu/commit/04beb290e4539f000aabe03651ce51e2e0f947ca.

**Comment by avinash-palleti** [comment]
@aguedes I added suggested changes and updated commit message also. Please check.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+sudo: required
+
+language: c
+# git submodules workaround
+git:
+  submodules: false
+# use sed to replace SSH URL with the public URL, then init submodules 
+before_install:
+  - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
+  - git submodule update --init --recursive
```
Comment (aguedes) [inline .travis.yml]: libavtp doesn't have any submodule so is this really required? Also, please review this whole file it looks we have more commands not required.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+dist: trusty
```
Comment (aguedes) [inline .travis.yml]: The following comment is related to the commit message:

Please replace the commit message by the message in the "original" commit https://github.com/AVnu/OpenAvnu/commit/76c6e71e2a56ba19b4527abebc24508d3e0bce76 in order to preserve the history.

Comment (avinash-palleti) [reply .travis.yml]: Done.

File: .travis.yml (hunk 1 above)
Comment (avinash-palleti) [reply .travis.yml]: Done. Reviewed packages in .travis.yml and updated.

**Comment by andrew-elder** [comment]
Looks ok to me. I'm going to merge.

**Pull request merged**